src/Sulu/Bundle/TestBundle/Behat/BaseContext.php

```diff
@@ -0,0 +1,354 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TestBundle\Behat;
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Behat\Symfony2Extension\Context\KernelAwareContext;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\Security\Core\SecurityContextInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+use Doctrine\Common\DataFixtures\Purger\ORMPurger;
+use Behat\MinkExtension\Context\MinkContext;
+use Symfony\Component\Console\Output\NullOutput;
+use Behat\MinkExtension\Context\RawMinkContext;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\StreamOutput;
+use Sulu\Bundle\SecurityBundle\Entity\User;
+
+/**
+ * Base context extended by all Sulu Behat contexts
+ * Note this context does not and should not contain any specifications.
+ * It is the base class of all Contexts.
+ */
+abstract class BaseContext extends RawMinkContext implements Context, KernelAwareContext
+{
+    const LONG_WAIT_TIME = 30000;
+    const MEDIUM_WAIT_TIME = 5000;
+
+    /**
+     * @var KernelInterface
+     */
+    protected $kernel;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setKernel(KernelInterface $kernel)
+    {
+        $this->kernel = $kernel;
+    }
+
+    /**
+     * Return the user ID.
+     *
+     * This currently could be any integer I believe
+     *
+     * @return integer
+     */
+    protected function getUserId()
+    {
+        return 1;
+    }
+
+    /**
+     * Execute a symfony command
+     * 
+     * $this->executeCommand('sulu:security:user:create', array(
+     *     'firstName' => 'foo',
+     *     '--option' => 'bar',
+     * ));
+     *
+     * @param string $command Command to execute
+     * @param array $args Arguments and options
+     *
+     * @return integer Exit code of command
+     */
+    protected function execCommand($command, $args)
+    {
+        $kernel = $this->kernel;
+
+        array_unshift($args, $command);
+        $input = new ArrayInput($args);
+
+        $application = new Application($kernel);
+        foreach ($kernel->getBundles() as $bundle) {
+            $bundle->registerCommands($application);
+        }
+
+        $application->setAutoExit(false);
+        $application->setCatchExceptions(false);
+        $command = $application->find($command);
+
+        $output = new StreamOutput(fopen('php://memory', 'w', false));
+        $res = $application->run($input, $output);
```

Comment:
Sry that I didn't already see that yesterday, but we don't use abbreviations ;-)